site/en/docs/privacy-sandbox/ip-protection/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'IP Protection'
+subhead: >
+  Improve user privacy by protecting their IP address from being used for tracking.
+description: >
+  A proposal to improve user privacy by protecting their IP address from being used for tracking. 
+date: 2022-12-01
+authors:
+  - anusmitaray
+---
+
+{% Aside %}
+This proposal used to previously have a different scope and was known as Gnatcatcher, or the Global Network Address Translation Combined with Audited and Trusted CDN or HTTP-Proxy Eliminating Reidentification. You can still view the [deprecated documentation](/docs/privacy-sandbox/gnatcatcher/).
+{% endAside %}
+
+## Implementation status
+
+This document outlines a new proposal to prevent covert tracking: IP Protection.
+
+*  The [IP Protection proposal](https://github.com/spanicker/ip-blindness) has entered [public discussion](https://github.com/spanicker/ip-blindness/issues).
+*  This proposal has not been implemented in any browser.
+*  [The Privacy Sandbox timeline](https://privacysandbox.com/open-web/#the-privacy-sandbox-timeline) provides implementation timings for other Privacy Sandbox proposals. IP Protection is still in its early phase and will have firmer timelines available on this website when available.
+
+## Why do we need this proposal?
+
+IP addresses were created to provide unique identifiers for a client so that traffic can be routed over the internet. IP addresses can be stable over periods of time, which can lead to user identification across first-parties.
```

Comment:
I wonder if this needs a little more explanation? I quite like the text I used (heavily reviewed from K folks!) on https://web.dev/digging-into-the-privacy-sandbox, so something like this: 

> A user's IP address is the public 'address' of their computer on the internet, which in most cases is dynamically assigned by the network through which they connect to the internet. However, even dynamic IP addresses may remain stable over periods of time, and this can lead to user identification across first parties.